src/main/java/io/appium/java_client/HasSessionDetails.java

```diff
@@ -17,15 +17,20 @@
 package io.appium.java_client;
 
 import static io.appium.java_client.MobileCommand.GET_SESSION;
+import static io.appium.java_client.MobileCommand.GET_ALLSESSION;
```

Comment:
Checkstyle will fail here. GET_ALLSESSION should go first.

Reply:
@SrinivasanTarget  will update, thanks